mteb/tasks/STS/kor/KlueSTS.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class KlueSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="KLUE-STS",
+        dataset={
+            "path": "klue",
+            "name": "sts",
+            "revision": "349481ec73fff722f88e0453ca05c77a447d967c",
+        },
+        description="",
+        reference="https://arxiv.org/abs/2105.09680",
+        type="STS",
+        category="s2s",
+        eval_splits=["validation"],
+        eval_langs=["kor-Hang"],
+        main_score="cosine_spearman",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
```

Comment:
Please fill these in according to https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md

Reply:
I was looking for information on how to enter metadata, thanks for the heads up. I added the form and domain by referencing the paper. I left the subtype as None because it doesn't apply.